src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2222,12 +2222,44 @@ public override BoundNode VisitWhileStatement(BoundWhileStatement node)
             return base.VisitWhileStatement(node);
         }
 
-        public override BoundNode VisitWithExpression(BoundWithExpression expr)
+        public override BoundNode VisitWithExpression(BoundWithExpression withExpr)
```

Comment:
>VisitWithExpression [](start = 34, length = 19)

Similar comment here, after Andy's latest PR, I think we should be able to re-use logic for object initializers.